content/docs/integrations/github-org/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Loading Github Organization Teams
+lastUpdated: '2021-03-19T12:14:39.0Z'
+description: How to configure backstage to read teams from github
+---
+
+## Introduction
+
+Roadie can load your organization team structure and employees from GitHub teams.
+
+This page describes how to do that.
+
+## Prerequisites
+
+- You will need a git repository for the configuration of Backstage. In the examples
+  below.
+- The github token configured in the backstage secrets must have the `read:user` and `user:email` scopes.
+
+## Steps
+
+1. If the name of your organization was "acme", you would create a file in the github repository called github-org.yaml with the following contents:
+
+```yaml
```

Comment:
The code block has different left margin than the text around it. Either remove the numbers from the steps around this code block or indent the code block so it is visually associated with the step it relates to.

Reply:
done